includes/ThemeSupport.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Theme support utility class.
+ *
+ * @package Google\Web_Stories
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories;
+
+/**
+ * Class ThemeSupport.
+ *
+ * @package Google\Web_Stories
+ */
+class ThemeSupport {
+
+	/**
+	 * Name for theme support.
+	 */
+	const NAME = 'web-stories';
+
+	/**
+	 * Get theme support and arguments for theme support.
+	 *
+	 * @return array
+	 */
+	public static function get_theme_support() {
```

Comment:
Where and how would this method be used?

Reply:
This would be used in Customizer and Stories' wrapper function.